README.md

```
@@ -10,315 +10,365 @@
 [![Commits](https://img.shields.io/github/commit-activity/m/glific/glific)](https://img.shields.io/github/commit-activity/m/glific/glific)
 [![Glific](https://img.shields.io/endpoint?url=https://dashboard.cypress.io/badge/simple/ocex65&style=flat&logo=cypress)](https://dashboard.cypress.io/projects/ocex65/runs)
 
-## Pre-requisites
+## Table of Contents
+
+* [Pre-requisites](#1-pre-requisites)
+   * [Software Dependencies](#software-dependencies)
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 3

(MD007, ul-indent)